README.md

```diff
@@ -233,6 +233,13 @@ optimism_package:
         # Offset is in seconds
         interop_time_offset: ""
 
+        # Whether to fund dev accounts on L2
+        # Defaults to True
+        fund_dev_accounts: true
```

Comment:
I feel like this could be a root argument, similar to batcher, see comment below.

Reply:
I'll let you decide this.

Reply:
I'm actually happy with both options :D

Reply:
root `chain` keys are assumed to contain dictionaries right now: https://github.com/ethpandaops/optimism-package/blob/05d0fe972f8096570432f8360902430e0490d619/src/package_io/sanity_check.star#L117-L121 

and adding this as root param would make it not possible to configure on per-chain basis

perhaps we keep it in `network_params`? it also feels semantically similar to other genesis-related arguments there

Reply:
friendly ping @barnabasbusa :) are you OK with keeping it as is?

Reply:
yes